Simplify PR template checklist (#1)
clearbanc/.github: .github/pull_request_template.md

## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>

## Checklist
- [ ] Add a brief description, screenshots (if applicable), and testing instructions for this PR.
- [ ] Introduce or modify relevant documentation

> - [ ] Ensure unhappy paths are considered and tested

Reviewer: ```suggestion
```

## Post-release Monitoring Plan <sub><sup>(what's your plan to monitor the changes to make sure they have the intended effect)</sup></sub>

## Checklist

> - [ ] Add a brief description of the purpose of this PR.
> - [ ] Add screenshots for this PR.
> - [ ] Add testing instructions for this PR.

Reviewer 2: ```suggestion
- [ ] Add a brief description, screenshots (if applicable), and testing instructions for this PR.
```

- [ ] Introduce or modify relevant documentation
- [ ] Ensure unhappy paths are considered and tested

> - [ ] Ensure there is a plan for monitoring post-deployment

Reviewer 2: ```- [ ] Ensure unhappy paths are considered/tested, and there is a plan for monitoring post-deployment```

Reviewer 2: consolidate the two WCGW and monitoring
